pytorch_lightning/trainer/connectors/logger_connector/fx_validator.py

```diff
@@ -97,22 +163,28 @@ class _LogOptions(TypedDict):
     }
 
     @classmethod
-    def check_logging(cls, fx_name: str, on_step: bool, on_epoch: bool) -> None:
+    def check_logging_and_get_default_levels(cls, fx_name: str, on_step: bool, on_epoch: bool) -> Tuple[bool, bool]:
```

Comment:
Then I'll have to split the old function as well because the check for on_step/on_epoch needs them to be boolean. So there will be 3 sequential calls then: check whether logging is supported, get default on_step/on_epoch, check whether on_step/on_epoch is supported.

Reply:
We can have all 3 and an utility to call the 3 together

Reply:
Okay! I'll break them then